flaml/automl/automl.py

```diff
@@ -2584,6 +2668,10 @@ def cv_score_agg_func(val_loss_folds, log_metrics_folds):
                 X_train = ray.get(X_train)
             elif isinstance(dataframe, ray.ObjectRef):
                 dataframe = ray.get(dataframe)
+        else:
+            self._state.resources_per_trial = (
+                {"cpu": n_jobs} if n_jobs > 0 else {"cpu": 1}
+            )
```

Comment:
Is this correct? Add "TODO" if you envision this to be changed in future.

Reply:
It's not used in spark for now. Added TODO.